### ISSUE TITLE
Double click on VisualFSharp.UnitTests test case does not go to the source code

### ISSUE THREAD

**Issue by KevinRansom** [dotnet/fsharp#13462]
Repro:
1. Sync and Build the Repo
2. Double click on any VisualFSharp.UnitTests test case
3. Observe it fail to display source code.

Expected, source code window to open with the cursor at the location of the test case.

**Comment by dsyme** [comment]
I fixed this  #13119 in the RFC work.  It's very odd.  By doing a binary chop I isolated it to this
```
    [<Ignore("Legacy test, NRE trying to get UI thread.")>]
    [<Test>]
    member public this.``ReferenceResolution.Bug650591.AutomationReference.Add.FullPath``() = 
```
Commenting out this test makes double-click work.  I've no idea why

**Comment by psfinaki** [comment]
I have further narrowed it down to
```fsharp
let invoker = 
    {
        new Microsoft.Internal.VisualStudio.Shell.Interop.IVsInvokerPrivate with
            member this.Invoke(invokable) = invokable.Invoke()
    }
```
Looking into that.
